commit bee15b629949bcfa3e0cedcb357534ef10c75998 (refs/changes/84/5584/1)
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2019-02-13T17:11:32-07:00 (8 months ago)
    
    MANTA-4117 Log files for pg_prefaulter should be stored in manta

diff --git a/boot/setup.sh b/boot/setup.sh
index 926ae76..0abb9e0 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -168,6 +168,12 @@ function manta_setup_manatee_env {
     manta_add_logadm_entry "manatee-snapshotter"
     manta_add_logadm_entry "postgresql" "/var/pg"
     manta_add_logadm_entry "pgdump" "/var/log/manatee"
+    # The following entry does not use the manta_add_logadm_entry function
+    # because the underscore in the pg_prefaulter name breaks the upload script
+    logadm -w pg_prefaulter -C 48 -c -p 1h \
+        -t "/var/log/manta/upload/pg-prefaulter_\$nodename_%FT%H:00:00.log" \
+        "/var/svc/log/*pg_prefaulter*.log" || fatal \
+        "unable to create logadm entry"
 }
 
 
